### PR TITLE
Ability to maximize map

### PR DESCRIPTION
**Pull request by Chessray**
Included a fullscreen component which allows maximizing the map views.

Closes [AB#1052](https://dev.azure.com/ncia-anet/2aa083a5-af3d-44e1-8c7b-6e9e6b124d91/_workitems/edit/1052)

#### User changes
- Users can set maps to full-screen and back

#### Superuser changes
- None

#### Admin changes
- None

#### System admin changes
- [ ] anet.yml or anet-dictionary.yml needs change
- [ ] db needs migration
- [ ] documentation has changed
- [ ] graphql schema has changed

### Checklist
  - [x] Described the user behavior in PR body
  - [x] Referenced/updated all related issues
  - [x] commits follow a `repo#issue: Title` title format and [these 7 rules](https://chris.beams.io/posts/git-commit/)
  - [x] commits have a [clean history](https://epage.github.io/dev/commits/), otherwise PR may be squash-merged
  - [ ] Added and/or updated unit tests
  - [x] Added and/or updated e2e tests
  - [ ] Added and/or updated data migrations
  - [ ] Updated documentation
  - [x] Resolved all build errors and warnings
  - [ ] Opened debt issues for anything not resolved here

### CODE REVIEW

File: client/package.json [hunk 1]
```diff
@@ -150,6 +150,7 @@
     "leaflet-defaulticon-compatibility": "0.1.2",
     "leaflet-geosearch": "3.11.1",
     "leaflet-gesture-handling": "1.2.2",
+    "leaflet.fullscreen": "^3.0.1",
```
Comment (gjvoosten) [inline client/package.json:153]: Please specifiy an exact version in package.json.

**Review by gjvoosten — CHANGES_REQUESTED** [review]

File: client/yarn.lock [hunk 2]
```diff
@@ -15410,7 +15417,16 @@ string-natural-compare@^3.0.1:
   resolved "https://registry.yarnpkg.com/string-natural-compare/-/string-natural-compare-3.0.1.tgz#7a42d58474454963759e8e8b7ae63d71c1e7fdf4"
   integrity sha512-n3sPwynL1nwKi3WJ6AIsClwBMa0zTi54fn2oLU6ndfTSIO05xaznjSf15PcBZU6FNWbmN5Q6cxT4V5hGvB4taw==
 
-"string-width-cjs@npm:string-width@^4.2.0", "string-width@^1.0.2 || 2 || 3 || 4", string-width@^4.1.0, string-width@^4.2.0, string-width@^4.2.3:
+"string-width-cjs@npm:string-width@^4.2.0":
```
Comment (gjvoosten) [inline client/yarn.lock:15420]: These changes seem to be unrelated to the new package leaflet.fullscreen. Why are they here?

Comment (Chessray) [reply client/yarn.lock:15420]: Good question. I simply ran `yarn add leaflet.fullscreen`.

**Review by pacodelcastillolopez — APPROVED** [review]

**Review by gjvoosten — APPROVED** [review]
